pagopa/io-app
# fix(IDPay): [IODPAY-189] Fix long names in IDPay initiative details screen

## Short description
This PR fixes the visualization of long initiative names in IDPay initiative details screen

| Before 	| After 	|
|---	|---	|
|  <img src="https://user-images.githubusercontent.com/6160324/231760876-e5a4fafa-f66b-431d-9181-af8646752665.png" width="200" />	|  <img src="https://user-images.githubusercontent.com/6160324/231760892-7b0a0e20-01a6-4a68-82a3-93b4f855228d.png" width="200" />	|




## List of changes proposed in this pull request
- Added correct styles to initiative name

## How to test
- With `io-dev-server`, create an initiative with a very long name. 
- From the wallet section in app, navigate to details screen of the initiative and check that the title is displayed correctly
